Create Tutor with Jest

CesarSimionato opened the pull request:
Eu e o Gian fizemos a implementação da rota de entrada e saída de dados para a criação do tutor, juntamente com a parte de testes com o jest da mesma.

Commit:
Paths and Jest

Commit:
Jest config paths

Commit:
create tutor test

andersonluisribeiro commented:
Pessoal, o PR de vocês está com problemas de build. Precisamos resolver primeiro antes da revisão.

Glatharth committed:
Downgrade of express version

A ultima versão do ExpressJS no NPM é a 4.17.1.
https://www.npmjs.com/package/express
http://expressjs.com/en/changelog/4x.html

CesarSimionato committed:
Merge pull request #1 from Glatharth/patch-1

Downgrade of express version
